pyodide_build/config.py

```diff
@@ -66,6 +66,7 @@ def _get_make_environment_vars(self) -> Mapping[str, str]:
             capture_output=True,
             text=True,
             env={"PYODIDE_ROOT": str(self.pyodide_root)},
+            check=True,
```

Comment:
The next line is to check the returncode...
```suggestion
            check=False,
```